web-dashboard/index.html

```
@@ -0,0 +1,1398 @@
+        <button class="qbtn" style="color:var(--sapphire);border-color:rgba(116,199,236,0.3)" onclick="agentCmd('apply')">apply</button>
+        <button class="qbtn" style="color:var(--sapphire);border-color:rgba(116,199,236,0.3)" onclick="agentCmd('batch')">batch</button>
+        <button class="qbtn" style="color:var(--sapphire);border-color:rgba(116,199,236,0.3)" onclick="agentCmd('deep')">deep eval</button>
+        <button class="qbtn" style="color:var(--sapphire);border-color:rgba(116,199,236,0.3)" onclick="agentCmd('patterns')">patterns</button>
+        <button class="qbtn" style="color:var(--sapphire);border-color:rgba(116,199,236,0.3)" onclick="agentCmd('training')">training</button>
+        <button class="qbtn" style="color:var(--sapphire);border-color:rgba(116,199,236,0.3)" onclick="agentCmd('pipeline')">pipeline</button>
+        <button class="qbtn" style="color:var(--sapphire);border-color:rgba(116,199,236,0.3)" onclick="agentCmd('project')">project</button>
+      </div>
+    </div>
+    <div class="terminal-input-row">
+      <span class="prompt">~/career-ops $</span>
+      <input class="term-input" id="term-input-full" placeholder="scan · pdf · apply · merge · tracker · help …" autocomplete="off" spellcheck="false">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)